website/docs/r/emrv2_cluster.html.markdown

```diff
@@ -178,7 +178,7 @@ resource "alicloud_emrv2_cluster" "default" {
 The following arguments are supported:
 
 * `resource_group_id` - (Optional) The Id of resource group which the emr-cluster belongs.
-* `payment_type` - (Optional, ForceNew) Payment Type for this cluster. Supported value: PayAsYouGo or Subscription.
+* `payment_type` - (Optional, Available modification since v1.226.0) Payment Type for this cluster. Supported value: PayAsYouGo or Subscription.
```

Comment:
(Optional)  XXXXX     **NOTE:** From version 1.226.0, `payment_type` can be modified.